src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -161,7 +162,7 @@ def find_date_in_string(string: str) -> Optional[str]:
     monthname_date_regex = rf"{lookbehind}({all_months}) ?{day_regex}, ?{year_regex}{lookahead}"
     monthname_date = regex.search(monthname_date_regex, string)
     # template: 1849/50 | 1849-50 | 1849/1850
-    year_range = regex.search(lookbehind + year_regex + r"[/-](\d{2}|\d{4})" + lookahead, string)
+    year_range = regex.search(lookbehind + year_regex + r"[/-](\d{2,4})" + lookahead, string)
```

Comment:
If you are working on that, could you not also build in:
1640-1
as in 1640-1641

Reply:
Done: https://github.com/dasch-swiss/dsp-tools/pull/652/commits/9311344f35e03c11a13f4463e47ae02c7f589320